site/src/modules/dashboard/DashboardProvider.tsx

```diff
@@ -23,20 +32,40 @@ export const DashboardContext = createContext<DashboardValue | undefined>(
 
 export const DashboardProvider: FC<PropsWithChildren> = ({ children }) => {
   const { metadata } = useEmbeddedMetadata();
+  const { user, organizationIds } = useAuthenticated();
   const entitlementsQuery = useQuery(entitlements(metadata.entitlements));
   const experimentsQuery = useQuery(experiments(metadata.experiments));
   const appearanceQuery = useQuery(appearance(metadata.appearance));
 
   const isLoading =
     !entitlementsQuery.data || !appearanceQuery.data || !experimentsQuery.data;
 
+  const lastUsedOrganizationId = localStorage.getItem(
+    `user:${user.id}.lastUsedOrganizationId`,
+  );
+  const [activeOrganizationId, setActiveOrganizationId] = useState(() =>
+    lastUsedOrganizationId && organizationIds.includes(lastUsedOrganizationId)
+      ? lastUsedOrganizationId
+      : organizationIds[0],
+  );
```

Comment:
Talked to @aslilac, we are not going to introduce the concept of "Default Org" to the FE. It is a stop gap on the backend to accommodate features like group sync, role sync, SCIM, etc. Those features need to be upgraded to not rely on default org.

If a deployment exists with more than 1 org, this is no more _broken_ than it currently is today.